app/src/main/java/com/nervousfish/nervousfish/activities/LoginActivity.java

```diff
@@ -19,29 +19,29 @@
 /**
  * Simple login activity class.
  */
-public final class LoginActivity extends Activity {
+public final class LoginActivity extends AppCompatActivity {
```

Comment:
The own one we use now has more flexibility to create for example the nicer title